WSRequestHandler.cpp

```diff
@@ -1093,4 +1095,286 @@ void WSRequestHandler::HandleGetRecordingFolder(WSRequestHandler* req)
 
 	req->SendOKResponse(response);
 	obs_data_release(response);
-}
\ No newline at end of file
+}
+
+void WSRequestHandler::HandleGetTextGDIPlusProperties(WSRequestHandler* req)
+{
+
+	const char* itemName = obs_data_get_string(req->data, "source");
+	if (!itemName)
+	{
+		req->SendErrorResponse("invalid request parameters");
+		return;
+	}
+
+	const char* sceneName = obs_data_get_string(req->data, "scene-name");
+	obs_source_t* scene = Utils::GetSceneFromNameOrCurrent(sceneName);
+	if (scene == NULL) {
```

Comment:
all of WSRequestHandler.cpp uses NULL plus that code is what was used in WSRequestHandler::HandleSetSceneItemRender line 322, 328 etc so I just copied it.

I but I'm happy to make it different 

![image](https://user-images.githubusercontent.com/5418596/27333040-7405f9de-5592-11e7-83c4-f5bf7b2a0ecc.png)

sorry I can't figure out how to do the text copy thing that you did